lib/new_relixir/plug/instrumentation.ex

```diff
@@ -1,15 +1,18 @@
 defmodule NewRelixir.Plug.Instrumentation do
   @moduledoc """
-  Utility methods for instrumenting parts of an Elixir app.
+  Utility methods for instrumenting parts of an Ecto app.
```

Comment:
module doc and the name of the module aren't really in agreement 🤔

Reply:
@coburncoburn Fair point. I avoided renaming the module now to keep fewer deprecations in this release, but we should be touching Ecto soon :+1:.